collections/ansible_collections/purestorage/flashblade/plugins/modules/purefb_fs_replica.py

```diff
@@ -229,9 +229,6 @@ def main():
     if not local_fs:
         module.fail_json(msg='Selected local filesystem {0} does not exist.'.format(module.params['name']))
 
-    if local_replica_link:
-        if local_replica_link.status == 'unhealthy':
-            module.fail_json(msg='Replca Link unhealthy - please check remote array')
```

Comment:
My concern here is that by removing this check condition, the module will continue and if the condition still exists when the module attempts to perform an action on the unhealthy link it will fail. 
It might be better to change the `fail_json` to call `warn`, thereby creating a message that the module may fail due to an unhealthy link.

Reply:
The only task we can perform after creating the Replica link is to attach and detach policies and that we can perform even on the unhealthy replica-link. Creating a warning message may confuse users.

Reply:
That is true today, but there is no guarantee that this won't change in the future.

Reply:
That may be true today, but there is no guarantee this will be true in the future